python/lsst/ts/watcher/rules/cp_verify_alarm.py

```diff
@@ -0,0 +1,266 @@
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+__all__ = ["CpVerifyAlarm"]
+
+import yaml
+import json
+import lsst.daf.butler as dafButler
+
+from lsst.ts.idl.enums.Watcher import AlarmSeverity
+from lsst.ts import watcher
+
+
+class CpVerifyAlarm(watcher.BaseRule):
+    """Monitor state of cp_verify executions from the OCPS
+    for a particular type of calibration (BIAS, DARK, FLAT, etc).
+
+    Set alarm severity NONE if the cp_verify tests passed,
+    and issue a SERIOUS alarm if not.
+
+    Parameters
+    ----------
+    config : `types.SimpleNamespace`
+        Rule configuration, as validated by the schema.
+
+    Notes
+    -----
+    The alarm name is f"CpVerifyAlarm.{calibration_type}:{ocps_index}",
+    where ``calibration_type`` and ``ocps_index`` are derived from
+    ``config.calibration_type`` and ``config.ocps_index``, respectively.
+    """
+
+    def __init__(self, config):
+        remote_name = "OCPS"
+        remote_index = config.ocps_index
+        remote_info = watcher.RemoteInfo(
+            name=remote_name,
+            index=remote_index,
+            callback_names=["job_result"],
+            poll_names=[],
+        )
+        super().__init__(
+            config=config,
+            name=f"CpVerifyAlarm.{remote_info.name}:{remote_info.index}",
+            remote_info_list=[remote_info],
+        )
+
+        self.calibration_type = config.calibration_type
+
+    @classmethod
+    def get_schema(cls):
+        schema_yaml = """
+            $schema: http://json-schema.org/draft-07/schema#
+            description: Configuration for CpVerifyBiasAlarm
+            type: object
+            properties:
+                calibration_type:
+                    description: >-
+                        Alarm calibration type: BIAS, DARK, or FLAT.
+                    type: string
+                ocps_index:
+                    OCPS index (e.g., OCPS:1: LATISS; OCPS:2: LSSTComCam).
+                verification_threshold:
+                    type: integer
+                    descriptor: Maximum number of failures per detector per test type.
+                default: 8
+            required: [name]
+            additionalProperties: false
+        """
+        return yaml.safe_load(schema_yaml)
+
+    def __call__(self, topic_callback):
+        msg = topic_callback.get()
+        # OCPS response after calling the pipetask
+        response = json.loads(msg.result)
+        # boolean: did verification fail?
+        verify_tests_pass = await self.get_verify_tests_pass_boolean(response)
```

Comment:
I think you need more information out of the `get_verify_test_pass_boolean` than a single boolean. Also, usually it is not a good practice to add the "return value"  to the name of the function.

I think you can return the value itself instead.. so basically this would become:

```
return await self.check_response(response)
```